tests/test_sample_sheet.py

```diff
@@ -325,6 +325,24 @@ def test_parse_different_length_header(self):
 
         assert_raises(ValueError, SampleSheet, filename)
 
+    def test_parse_limited_commas(self):
+        """Test minimium required commas"""
+        filename = string_as_temporary_file(
+            '[Header]\n'
+            'IEMFileVersion,4\n'
+            'Description\n'
+            'Chemistry,Default\n'
+            '[Settings]\n'
+            '\n'
+            '[Reads]\n'
+            '\n'
+            '[Data]\n'
+            'Sample_ID, Description\n'
+            'test2, Sample Description\n'
+        )        
+        SampleSheet(filename)
```

Comment:
Thanks for the test @golharam! Could you have a few assertions that parsing was correct? All you're testing at the moment is that you can build a `SampleSheet` instances with one parameter. I may, someday, implement lazy parsing so this test would be useless.

What I would do is assert that the sample sheet contains your expected sample, and that the description field is empty.

Reply:
I thought this test would be sufficient, since the current failure is that the sample sheet cannot be parsed at all.  I'll add some checks that the expected samples are there and the description field is empty...